app/test_split_integer.py

```diff
@@ -2,20 +2,35 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 22
+    parts = 4
+    result = split_integer(value, parts)
+    assert sum(result) == value
```

Comment:
Based on clean code recommendations, will be better to do:

```suggestion
    assert sum(split_integer(22, 4)) == 22
```
or
```suggestion
    assert split_integer(22, 4) == [4, 4, 4, 6]
```
![image](https://github.com/mate-academy/py-split-integer/assets/87720583/49a02dc8-9014-49c8-a9e2-9ab868268fa1)